src/app/zones/views/ZoneListView.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <RouteView
-    v-slot="{ route }"
+    v-slot="{ route, t, can }"
```

Comment:
**Question**: Remind me, why do you access `t` and `can` via `RouteView`? Both of them should be available as utilities via a `const { t } = useI18n()` and `const can = useCan()`, no?

Reply:
They are things you will pretty much _always_ need in a route component, ideally they should never be used outside of a route component.

Reply:
Why? There doesn’t seem to be a good reason to impose such a limitation and exposing this utility in the script just seems easier. Also, it’s quite likely that these utilities will be necessary in non-component code (for example on the route level).

Reply:
> Also, it’s quite likely that these utilities will be necessary in non-component code (for example on the route level).

It's fine to use these in non-component code, just inject the service where necessary.

When I said "ideally they should never be used outside of a route component", I meant "ideally they shouldn't be used in non-route components" i.e. I was only considering components here, and when I say "ideally" I mean there are obviously times when you do want to use these in non-route components, but its far less common, and using them in the route component and passing the data down should be the 'default'.

Reply:
So then why don’t we just always access `can` via the script section?

Reply:
We can't we just always access `can` via the template section? i.e. why have an entire script section plus `import`, plus `useCan` if we don't even need a script section otherwise?

Umm, could you explain more what your issue is with using:

```javascript
RouteView = (can) => {
  can('')
}
```

instead of:

```javascript
import '@/app/.../can'
const can = useCan()
RouteView = () => {
  can('')
}
```

Reply:
I just don’t understand why `RouteView` has anything to do with `can`. It’s incidental that  that component appears in commonly.